hieradata/dev/wso2/wso2is/pattern-1/default.yaml

```diff
@@ -58,6 +57,8 @@ wso2::template_list:
 #  - repository/conf/security/cipher-tool.properties
 #  - repository/conf/security/cipher-text.properties
 #  - repository/conf/security/secret-conf.properties
+
+#wso2::file_list_copy_without_refresh:
```

Comment:
+1 This can be confusing to a user without inside knowledge of why such a variable was needed.